scripts/utils/generate_ts.sh

```diff
@@ -113,6 +128,27 @@ for branch in $(git branch -r | grep origin/releases); do
   done
 done
 
+# When creating translation files for l10n repo, remove any addon-specific files that use
+# shared strings, as they are translated via `strings.yaml`, which is set up for translation in
+# `build.py` (which like the addon ts files, is created above when addon's cmake calls `build.py`)"
+if [ $KEEP_ALL_TS_FILES == 0 ]; then
+  print Y "Checking for .ts files using shared strings"
+  for ts_file in ./addon_ts/*
+  do
+    # Use the addon name from the .ts file to build a path to the manifest
+    manifest_file="./addons/$(basename $ts_file .ts)/manifest.json"
+    if [[ -f $manifest_file ]]; then
+      uses_shared_strings=$(jq '.message.usesSharedStrings // false' < $manifest_file)
+      if [[ "$uses_shared_strings" == "true" ]]; then
+        print G "Deleting file because the addon uses shared strings: $ts_file"
+        rm $ts_file
+      fi
+    else
+      die "Manifest file not found: $manifest_file"
```

Comment:
We can't die here, given the whole "we extract string for addons from all branches". An addon is bound to be missing here and there :-( 


See https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/actions/runs/12166789612/job/33933835196

```
Checking for .ts files using shared strings
Manifest file not found: ./addons/message_try_firefox_extension/manifest.json
```

Reply:
I discovered this as well, have a fix I'm putting up shortly